index.css

```diff
@@ -222,10 +222,20 @@ h2{
 	text-align: center;
 }
 
+@media only screen and (max-width:1140px){
+	main{
+		max-width: calc(var(--max-content-width)*(2/3));	
+		align-items: center;
+		margin:0 auto;
+	}
+
+}
+
 @media only screen and (max-width: 746px) {
```

Comment:
I can't fit more than one column at 760 px yet this doesn't trigger until 746 px. Creating a range where the sections are not centered. You just can't use pixels here. That's just not going to work. The page is sized in em so you have to use em here.